### ISSUE TITLE
Name tools "Polaris for [Platform]"

### ISSUE THREAD

**Issue by martenbjork** [Shopify/polaris-react-archive#5245]
### TLDR
- Rename the VS Code Plugin to `Polaris for VS Code`
- Rename the Figma Plugin to `Polaris for Figma`
- Rename `#temp-tooling-toolbox-plugin` to `#temp-polaris-tools`

Any upcoming features will be bundled within these 2 plugins.

### Why?
- At Shopify, folks are always busy. Cutting through the noise is hard — even when offering an attractive tool. Therefore, the biggest tooling challenge is getting people to actually press the "install" button.

- At Shopify, there are a million resources. We want to make it easy for crafters to know that they have all the right Polaris tooling.

- Once we ship more and more features, our plugins will feel increasingly essential which will be good for adoption. (In contrast, if we ship many OK:ish plugins, neither will feel essential and folks may choose not to install any of them.)

- Less overhead when maintaining and publishing plugins.

### Feature opt-out
We don't want to force anyone to use a feature that they don't like. Therefore, we should consider adding "opt out" settings down the road, especially for `Figma for VS Code`. The settings would allow you to disable any features you don't like.

### Trip wires to watch out for
- Maintaining the architecture of respective plugins becomes hard
- Users tells us that the plugins feel over baked or bloated

### Vision
In the future, our website should link to these tools front and center. You should immediately be able to understand which tools are available for you.

![Desktop - 5 (4)](https://user-images.githubusercontent.com/875708/157450785-bf7eb130-5a6e-422d-a715-90300ed8ad9f.jpg)

**Comment by jjgali** [comment]
+1 on the naming. I think it's clear and a much better use of the "Polaris for..." phrase than what we tried to do with design systems (Polaris for retail, Polaris for billing, etc.). 👏 thanks, @martenbjork!
